cylc/flow/task_job_mgr.py

```diff
@@ -947,38 +947,29 @@ def _run_job_cmd(
             )
 
     @staticmethod
-    def _set_retry_timers(itask, rtconfig=None, retry=True):
+    def _set_retry_timers(
+        itask: 'TaskProxy',
+        rtconfig: Optional[dict] = None
+    ) -> None:
         """Set try number and retry delays."""
         if rtconfig is None:
             rtconfig = itask.tdef.rtconfig
-        if (
-            itask.tdef.run_mode + ' mode' in rtconfig and
-            'disable retries' in rtconfig[itask.tdef.run_mode + ' mode']
-        ):
-            retry = False
```

Comment:
I think this is left behind from an old configuration we removed at Cylc 8?

Reply:
according  to the Cylc 7 code these were obseleted at 7.2.2